web/src/cypress/integration/scheduleCalendar.ts

```diff
@@ -161,7 +171,12 @@ function testCalendar(screen: ScreenFormat) {
       .trigger('mouseover')
     cy.get('div[data-cy="shift-tooltip"]').should('be.visible')
     cy.get('button[data-cy="remove-override"]').click()
-    cy.get('button[type="submit"]').click()
-    cy.get('button[type="submit"]').should('not.exist')
+    cy.get('div[role=dialog]').as('dialog')
```

Comment:
Come to think of it, I think click returns the element anyhow, so we could probably wrap it all together and drop the alias.

Maybe try:
```js
cy.get('div[role=dialog]').contains('button','Submit').click().should('not.exist')
```